api/api/settings.py

```diff
@@ -37,7 +37,9 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    "cotrip_app",
+    'cotrip_app',
```

Comment:
What is the `cotrip_app`? Do we need it?

Reply:
it is where the location and city are going to go, but ill delete it on this PR!!! se can merge when those get a PR